DOCUMENTATION.md

```diff
@@ -216,10 +216,17 @@ You can leave out this argument if your service only has one reply-to email addr
 
 To send a file by email, add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file.
 
-The file will be available for the recipient to download for 18 months.
-
 The links are unique and unguessable. GOV.UK Notify cannot access or decrypt your file.
 
+Your file will be available to download for a default period of 78 weeks (18 months). From 29 March 2023 we will reduce this to 26 weeks (6 months) for all new files. Files sent before 29 March are not affected.
```

Comment:
Very small one that is optional as maybe our documentation doesn't need to be 100% precise necessarily.

Is it beneficial to say the default period is 78 weeks. Firstly, I'm guessing it may not always be exactly 78 weeks depending on how many weeks in a month? Secondly, at this point is it useful to make them start thinking in terms of weeks?

Same with the 26 weeks (6 months).

@samuelhwilliams Here is a slightly more interesting one though, what will our default retention period actually be when we change it. Will it be 26 weeks or will it be 6 months? Those could be very slightly different by a few days.

Reply:
It'll be 26 weeks. Everything will be in weeks - which I think is why we're starting to get them to think that way by how we speak in the docs.

Reply:
At the moment, documents in the S3 bucket expire after 546 days, which is 78 weeks. So that is correct.

Reply:
Thanks both, I’m resolving this.